### ISSUE TITLE
Plotting range of Learning Rate 

### ISSUE THREAD

**Issue by kanzaali** [bckenstler/CLR#24]
Hi, 
Thank you so much for your work.
I want to plot the range of learning rate base_lr= 0.01, max_lr=0.1 for my method 

`schedulers = torch.optim.lr_scheduler.CyclicLR(optim_backbone, base_lr= 0.01, max_lr=0.1, step_size_up=2000, step_size_down=None, mode='triangular')`

Like Figure 2(a) of this paper https://arxiv.org/pdf/1708.07120.pdf
